src/models/offer.js

```
@@ -3,45 +3,69 @@
 const {
   enums: { MAIN_ROLE_ENUM, SPOKEN_LANG_ENUM, PROFICIENCY_LEVEL_ENUM, OFFER_STATUS_ENUM }
 } = require('~/consts/validation')
-const { USER, OFFER } = require('~/consts/models')
+const { USER, OFFER, CATEGORY, SUBJECT } = require('~/consts/models')
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)